docs/en/latest/plugins/proxy-rewrite.md

```diff
@@ -57,67 +57,65 @@ Header configurations are executed according to the following priorities:
 The example below enables the `proxy-rewrite` Plugin on a specific Route:
 
 ```shell
-curl http://127.0.0.1:9180/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
-{
+curl "http://127.0.0.1:9180/apisix/admin/routes" -X PUT \
+  -H "X-API-KEY: ${ADMIN_API_KEY}" \
```

Comment:
Not sure if this should be copied from API7 docs

Reply:
ADMIN_API_KEY is actually edd1c9f034335f136f87ad84b625c8f1, so I think there is no problem

Reply:
I wrote the API7 docs and it's meant to be a user defined env var 😅